### ISSUE TITLE
JSON RPC field "num_insertions" does not persist if server is restarted

### ISSUE THREAD

**Issue by tgoodyear** [stingergraph/stinger#214]
The field `num_insertions` is not preserved when a graph is written to disk. It is instead reset to 0.

**Comment by jpfairbanks** [comment]
Where would you like to store that information? with the graph on disk or in an external store? a second file?

**Comment by ehein6** [comment]
It's right there in the main struct, seems like it ought to be with the graph on disk: https://github.com/stingergraph/stinger/blob/20e4b46446d86b4e1ef560618886ea4bcc73984d/lib/stinger_core/inc/stinger_internal.h#L128

**Comment by jpfairbanks** [comment]
yeah, unless the graph is being written to some other format like edge list or CSR.
